src/format.ts

```diff
@@ -92,7 +107,8 @@ function fixFormat(srcFiles: string[], baseArgs: string[]): Promise<boolean> {
  *
  * @param srcFiles list of source files
  */
-function checkFormat(srcFiles: string[], baseArgs: string[]): Promise<boolean> {
+function checkFormat(srcFiles: string[], baseArgs: string[], options: Options):
```

Comment:
Nit: Can we emulate `format` and put the `options` argument first?

Reply:
done